phc-rails/app/controllers/api/v1/event_registrations_controller.rb

```diff
@@ -3,36 +3,33 @@ class Api::V1::EventRegistrationsController < ApplicationController
   before_action :verify_security_token
 
   def create
-    event_reg = EventRegistration.new
-
-    # Create account if salesforce id was not passed in
-    sf_id = params[:account_sfid]
-    if (sf_id.nil? || sf_id.empty?)
-      # Create account of the parameters passed in, sf_id will be nil
-      params[:sf_id] = params.delete :account_sfid
+    sf_id = params.delete :account_sfid
+    if sf_id.nil? || sf_id.empty? || sf_id = "\"\""
+      # Create account if salesforce id was not passed in
+      # TODO: Right now, the client basically creates accounts through this endpoint
+      # We should seperate it into 2 API calls, the first to api/v1/create (which is unused right now)
+      params[:sf_id] = ""
       account = Account.spawn(params)
-
-      # Since we don't know the SF id yet, use the unique rails id for now to identify the account
-      event_reg[:account_sfid] = account.id
     else
       # We retrieve the corresponding account if the salesforce id was passed in
       if not Account.exists?(sf_id: sf_id)
         raise "Unknown Salesforce ID. This should not happen!"
       end
       account = Account.find_by(sf_id: params[:account_sfid])
-      event_reg[:account_sfid] = account.sf_id
     end
 
-    # Number__c is the QR_code that was passed in, which identifies a
-    # particular account for the current PHC event
```

Comment:
Is this comment now irrelevant? It looks like we're still using `Number__c` and since the format of that is strange I think we'd still like to know what it's doing.